### PR TITLE
Update sln to use package references

### LINKED ISSUE #125: Move to new csproj structure and start using package references.
- move to new csproj.
- update sln to VS 2019
- update to use package references.

original Wox has started doing the same -> https://github.com/Wox-launcher/Wox/pull/2680 use for reference

### PR DESCRIPTION
**Pull request by theClueless**

### CODE REVIEW

File: Wox.Plugin/BaseModel.cs [hunk 1]
```diff
@@ -1,11 +1,9 @@
 ﻿using System.ComponentModel;
 using System.Runtime.CompilerServices;
 using JetBrains.Annotations;
-using PropertyChanged;
 
 namespace Wox.Plugin
 {
-    [ImplementPropertyChanged]
```
Comment (jjw24) [inline Wox.Plugin/BaseModel.cs:8]: @theClueless is this change intended?

Comment (theClueless) [reply Wox.Plugin/BaseModel.cs:8]: yes, seems to work without it :)

**Review by jjw24 — APPROVED** [review]
